tests/post_training/test_quantize_conformance.py

```diff
@@ -104,6 +105,22 @@ def test_ptq_quantization(test_case_name, data, output, result):
             )
 
     run_info.time_total = time.perf_counter() - start_time
+    process_connection.send((run_info, err_msg))
+
+
+@pytest.mark.parametrize("test_case_name", TEST_CASES.keys())
+def test_ptq_quantization(test_case_name, data, output, result):
+    main_connection, process_connection = Pipe()
+
+    # Use multiprocessing to clean torch cache after trace model to convert in IR
```

Comment:
What exactly is meant as a "mem leak", and how exactly does using multiprocessing help in this case? If the culprit is known to be torch, why could this have not been fixed by invoking torch-specific functions for cache clearing?

Reply:
https://en.wikipedia.org/wiki/Memory_leak

You can reproduced it by 
```python
dummy_tensor = torch.rand([1, 3, 224, 224])
for i in range(42):
    timm_model = timm.create_model("hrnet_w18", num_classes=1000, in_chans=3, pretrained=True, checkpoint_path="")
    ov_model = ov.convert_model(timm_model, example_input=dummy_tensor)
```
It helps because torch cache cleaned on removing `Process`. 
Pytorch does not have public methods to clean cache like `torch.cuda.empty_cache()`

Reply:
So maybe you should wait until reviews will be requested?

Just for interesting, how memory leaks depends from size of RAM?

Reply:
> So maybe you should wait until reviews will be requested?
> 
> Just for interesting, how memory leaks depends from size of RAM?

I am automatically requested for review as a member of nncf-maintainers. If the PR is not ready for review, you should have left it in draft state.

Memory leaks depend on the size of RAM in a sense that the more RAM you have, the more time it takes for a leak to crash a process. The 42 iterations in your script weren't enough in my case to produce a crash, neither did they lead to GPU memory increase, so had I not observed the resident memory increase in `htop` in parallel, I could have missed the entire symptom.

Reply:
Now that you have introduced actual, specific code that solves the problem, the solution has become much better and it is no longer necessary to add extended explanations in-code (although it wouldn't hurt too).